bin/titan/main.go

```diff
@@ -52,6 +55,9 @@ func main() {
 		fmt.Println("Warning: Titan is running in memory mode, all the data would be cleared after the process exit.")
 		fmt.Println("The memory mode can only be used for experience, configure the tikv.pd-addrs to use the TiKV as a backend")
 	}
+	if dumpRedisCommand != "" {
```

Comment:
the same

Reply:
if `dumpRedisCommand` both setting on configuring file and cmd param. the cmd param will replace configuring file.